components/Tabs/examples/MDCTabBarViewTypicalExampleViewController.m

```diff
@@ -84,6 +84,7 @@ - (void)viewDidLoad {
                         forState:UIControlStateSelected];
   self.tabBar.selectedItem = item1;
   self.tabBar.translatesAutoresizingMaskIntoConstraints = NO;
+  self.tabBar.edgeInsets = UIEdgeInsetsMake(0, 120, 0, 20);
```

Comment:
Please pare these down to something a bit smaller. Perhaps `{0, 10, 0, 10}`?

Reply:
Removed.